### ISSUE TITLE
Nan in Photons

### ISSUE THREAD

**Issue by mariadalfonso** [cms-nanoAOD/cmssw#566]
we have many nan in photons nanoAOD-V8

root://xrootd-cms.infn.it//store/data/Run2016G/MET/NANOAOD/UL2016_MiniAODv1_NanoAODv2-v1/270000/4D848321-6D1D-F949-A95E-D4A8C263906D.root
> run:lumi:event 278820:518:939276241

root://xrootd-cms.infn.it//store/data/Run2016G/MET/NANOAOD/UL2016_MiniAODv1_NanoAODv2-v1/270000/CE684FF8-BA3F-244B-A841-661D88AE3163.root
> run:lumi:event 278820:328:587668886

root://xrootd-cms.infn.it///store/data/Run2016G/MET/NANOAOD/UL2016_MiniAODv1_NanoAODv2-v1/280000/59021744-BDFC-3C4B-AD8F-78EC96AD2F0F.root
> run:lumi:event 280007:8:8839035
> run:lumi:event 280007:8:9609024
> run:lumi:event 280007:8:8691064

Also the size of photons is peculiar. in some event of 59021744-BDFC-3C4B-AD8F-78EC96AD2F0F.root we have clusters of 500-700 photons

I have seen this in both data and QCD/ttbar MC

**Comment by mariadalfonso** [comment]
@sroychow @jainshilpi

please have a look, we need a fix for v9

**Comment by jainshilpi** [comment]
@mariadalfonso 

About the events here:
root://xrootd-cms.infn.it///store/data/Run2016G/MET/NANOAOD/UL2016_MiniAODv1_NanoAODv2-v1/280000/59021744-BDFC-3C4B-AD8F-78EC96AD2F0F.root

run:lumi:event 280007:8:8839035
run:lumi:event 280007:8:9609024
run:lumi:event 280007:8:8691064


We looked at one of the events run:lumi:event 280007:8:8839035
and the event looks like this in the fireworks:
https://cernbox.cern.ch/index.php/s/BYLmBjeYOlMhS8p

Checking with ECAL DPG (Fabrice, Simone and David), it seem that ECAL was bad in this run. Also in the central JSON (https://cms-service-dqmdc.web.cern.ch/CAF/certification/Collisions16/13TeV/Legacy_2016/Cert_271036-284044_13TeV_Legacy2016_Collisions16_JSON.txt), this run is not present. 

We will confirm about the nan problem by tomorrow.

**Comment by vlimant** [comment]
part of cleanup issue : @jainshilpi , has this issue been handled?

**Comment by jainshilpi** [comment]
Hi @vlimant, at that we scanned those events and we did not find any nan values - neither in isolation variables nor in any of the shower shape variables. It was also later checked by @mariadalfonso and she also confirmed about that. So I suppose this is already handled. But I let @mariadalfonso comment on this. If there are further problems, then I suppose this can be addressed to the current e/gamma POG conveners. Thanks, Shilpi
